examples/3d/perspective_zoom.rs

```diff
@@ -0,0 +1,125 @@
+//! Shows how to zoom and orbit a perspective projection camera.
+
+use std::ops::Range;
+
+use bevy::{input::mouse::AccumulatedMouseScroll, prelude::*};
+
+#[derive(Debug, Default, Reflect, Resource)]
+struct CameraSettings {
+    // Multiply keyboard inputs by this factor
+    pub orbit_speed: f32,
+    // Clamp fixed vertical scale to this range
+    pub zoom_range: Range<f32>,
+    // Multiply mouse wheel movements by this factor
+    pub zoom_speed: f32,
+}
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .init_resource::<CameraSettings>()
+        .add_systems(Startup, (setup, instructions))
+        .add_systems(Update, camera_controls)
+        .register_type::<CameraSettings>()
```

Comment:
```suggestion
```

Reply:
Ah, yep I was just reacting to Alice's point about tooling.